ipn/ipnlocal/local.go

```diff
@@ -3108,7 +3108,7 @@ func (b *LocalBackend) FileTargets() ([]*apitype.FileTarget, error) {
 	defer b.mu.Unlock()
 	nm := b.netMap
 	if b.state != ipn.Running || nm == nil {
-		return nil, errors.New("not connected")
+		return nil, errors.New("not connected to the Tailnet")
```

Comment:
Please lower-case "tailnet."

"Tailscale" is a name and should be capitalized. We consider _tailnet_ to be a regular noun.

Reply:
Done.